src/checker.rs

```diff
@@ -3,7 +3,7 @@ use std::error::Error;
 use reqwest::{Response, StatusCode};
 
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
-pub enum CheckResult {
+pub enum UrlCheckResult {
```

Comment:
Want to actually drop these changes from your patch? It might be easier if I merge mine first actually